### DISCUSSION TITLE
Data export

### DISCUSSION THREAD

**Discussion by adk7712** [SrNightmare09/chess-fen-reader#13] [Code Discussion]
discussion about data export

**Comment by iamitiwari** [comment]
Well sorry for the confusion, I thought some part of project is based on node.js. So I thought exporting current configuration(and probably past as well) of the board to .json file would be good option. This could also potentially allow for history bar kind of thing for FEN strings user has entered. But for now it is not needed.

**Reply by adk7712** [reply]
i see, ok then.
